simulator/travis/minimesosFile

```diff
@@ -132,18 +132,18 @@ minimesos {
 
             ports {
                 role = "*"
-                value = "[31000-32000]"
+                value = "[31400-31500]"
             }
             ports {
-                role = "*"
-                value = "[32000-33000]"
+                role = "cook"
```

Comment:
Is the "*" on line 44 not sufficient? Do you want both of the "ports" blocks for this agent to apply to the wildcard role?